tests/e2e-openshift-serverless/otel-tempo-serverless/chainsaw-test.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
```

Comment:
I'm not sure if we really need this E2E test because the OpenTelemetry Collector is just forwarding the traces. It should be enough with the direct Tempo integration.

Reply:
The Serverless docs document both scenarios. https://docs.openshift.com/serverless/1.32/observability/tracing/serverless-tracing.html .I have added both to detect any regression if anything changes for the components used.